raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -236,4 +176,31 @@ private static Integer parseVoterId(String idString) {
         return voterMap;
     }
 
+    public static class ControllerQuorumVotersValidator implements ConfigDef.Validator {
+        @Override
+        public void ensureValid(String name, Object value) {
+            if (value == null) {
+                throw new ConfigException(name, null);
+            }
+
+            @SuppressWarnings("unchecked")
+            List<String> voterStrings = (List) value;
+
+            if (voterStrings.size() == 0) {
+                // TODO: Add a flag to skip validation for an empty voter string, conditionally.
+                //       For now, skip anyway. See https://github.com/apache/kafka/pull/9916#discussion_r560611932
```

Comment:
We typically do not leave TODOs in the code. We can file a jira if we think it's important to remember. I'd suggest we just leave this check out and skip the empty check below.

Reply:
Fair enough. Removed.